examples/fall_detection/demo.ipynb

```diff
@@ -0,0 +1,323 @@
+    "                keypoints_list.append(keypoint)\n",
+    "            if len(keypoints_list) >= 30:\n",
+    "                break\n",
+    "        if len(keypoints_list) < 30:\n",
+    "            return 'unknown'\n",
+    "        # type conversion to adapt data to model input\n",
+    "        pts = np.array(keypoints_list, dtype=np.float32)\n",
+    "        out = Person.action_model.predict(pts, self._ctx.frame.shape[:2])\n",
+    "        action_name = Person.action_model.class_names[out[0].argmax()]\n",
+    "        return action_name"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Query on `Person`'s pose\n",
+    "\n",
+    "To filter on people that are falling down, we filter on `pose` having value `\"Fall Down\"` (7 actions should be supported: `\"Standing\", \"Walking\", \"Sitting\", \"Lying Down\", \"Stand up\", \"Sit down\", \"Fall Down\"`).\n",
+    "\n",
+    "`filter_cons` is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filter_cons = {\n",
+    "    '__class__': lambda x: x == Person,\n",
+    "    'pose': lambda x: x == \"Fall Down\"\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For output, we select:\n",
+    "\n",
+    "- tracker id, selected with `track_id`\n",
+    "- bounding box, in format of coordinates of top-left and bottom-right corner, selected with `tlbr`. Need to be converted to `str` before serializing.\n",
+    "\n",
+    "`select_cons` is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "select_cons = {\n",
+    "    'track_id': None,\n",
+    "    'tlbr': lambda x: str(x)\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The query could be:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class FallDetection(vqpy.QueryBase):\n",
+    "    @staticmethod\n",
+    "    def setting() -> vqpy.VObjConstraint:\n",
+    "        return vqpy.VObjConstraint(\n",
+    "            filter_cons=filter_cons,\n",
+    "            select_cons=select_cons,\n",
+    "            filename='fall'\n",
+    "        )"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running the query\n",
+    "\n",
+    "With the `Person` VObj and the query defined, we can run the query, with:\n",
+    "\n",
+    "- `cls_name` is a tuple for mapping numerical outputs of object detector to str\n",
+    "\n",
+    "\te.g. `vqpy.COCO_CLASSES` here starts with `(\"person\", \"bicycle\", ...)`, meaning that we will map output `0` of object detector to COCO class `\"person\"`\n",
```

Comment:
change the doc to that in `unattended baggage`

Reply:
Changed